docs/Configuration.md

```diff
@@ -147,41 +135,35 @@ oc create secret generic snow-secret \
 ```
 
 ### Labels
+Labels are key/value pairs that are attached to objects such as pods. They are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users and Pelorus.
 
-Labels are key/value pairs that are attached to objects, such as pods. Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users and Pelorus.
-
-The commit time, deploy time, and failure exporters all rely on labels to indentify the application that is associated with an object.  The object can
-include a build, build configuration, deployment or issue.
+The commit time, deploy time, and failure exporters all rely on labels to identify the application that is associated with an object.  The object can include a build, build configuration, deployment, or issue.
 
-In Pelorus the default label is: `app.kubernetes.io/name=<app_name>` where 
-app_name is the name of the application(s) being monitored. The label can
-be customized by setting the `APP_LABEL` variable to your custom value. 
+The default Pelorus label is: `app.kubernetes.io/name=<app_name>` where
+`app_name` is the name of the application(s) being monitored. The label can be customized by setting the `APP_LABEL` variable to a custom value.
 
-An example may be to override the APP_LABEL for the failure exporter to indicate a production bug or issue.
-The `APP_LABEL` with the value `production_issue/name` may give more context than `app.kubernetes.io/name`
-In this case the Github issue would be labeled with `production_issue/name=todolist`
+An example may be to override the APP_LABEL for the failure exporter to indicate a production bug or issue. The `APP_LABEL` with the value `production_issue/name` may give more context than `app.kubernetes.io/name`. In this case, the GitHub issue would be labeled with `production_issue/name=todolist`.
 
 Example Failure exporter config:
 ```
 - app_name: failure-exporter
   exporter_type: failure
   env_from_secrets:
-  - github-secret
+  - GitHub-secret
   extraEnv:
   - name: LOG_LEVEL
     value: DEBUG
   - name: PROVIDER
-    value: github
+    value: GitHub
```

Comment:
definitely small, or we will need to update code to match the PROVIDER value to ignore capital letters.